Cornelica/scenario/001.txt

```diff
@@ -6,11 +6,11 @@
 @bgm f=s-001
 
 @mes_win back=1 pos=1
-\CL\{淫魔\}-サキュバス-とは、
+\CL\A \{Succubus\} is,
 
-\CL\人の\{精\}を\{啜り\}生きる、\{異形\}の存在である
+\CL\a \{fantastical\} being that lives by \{sucking\} people's \{life energy\}.
```

Comment:
`\CL\a \{fantastical\} being that lives by \n\{sucking\} people's \{life energy\}.`